addons/packages/sriov-network-device-plugin/3.3.2/README.md

```diff
@@ -0,0 +1,43 @@
+# Sriov-Network-Device-Plugin Package
+
+This package contains Kubernetes device plugin for discovering and advertising SR-IOV virtual functions (VFs) available on a Kubernetes host. Using [sriov-network-device-plugin](https://github.com/k8snetworkplumbingwg/sriov-netowrk-device-plugin/).
+
+## Components
+
+* SRIOV Network Device Plugin DaemonSet
+* SRIOV Network Device Plugin ConfigMap
+* SRIOV Network Device Plugin ServiceAccount
+
+## Configuration
+
+The following configuration values can be set to customize the sriov-network-device-plugin installation.
+
+### Global
+
+| Value       | Required/Optional | Description                                                   |
+| ----------- | ----------------- | ------------------------------------------------------------- |
+| `namespace` | Required          | The namespace in which to deploy sriov-network-device-plugin. |
+
+### sriov-network-device-plugin DaemonSet Configuration
+
+| Value                   | Required/Optional | Description                                                                                                     |
+| ----------------------- | ----------------- | --------------------------------------------------------------------------------------------------------------- |
+| `daemonset.resources`             | Required          | The resources daemonset settings including resources. |
+| `daemonset.args`             | Required          | The args passed into daemonset's container. |
+| `sriov_nodes_resources` | Required          | The SR-IOV resource list sriov-network-device-plugin daemonset refers to.  |
+
+## Usage Example
```

Comment:
It would be great if we can add some introduction about resource list structure like key:value, resourceList for each node pool matched with label key:value here.

Reply:
Adding a new example section about this field.